File: openapi/code_samples/csharp/giftcards/order_a_gift_card/code.cs
using System;
using System.Threading.Tasks;
using System.Net.Http;
using System.Text;
using Newtonsoft.Json;

namespace WebAPIClient {
  class Program {
    static async Task Main(string[] args) {
      await ApiCall();
    }

    private static async Task ApiCall() {
      var json = JsonConvert.SerializeObject(new {
          productId = "120",
          countryCode = "US",
          quantity = "1",
          unitPrice = "1",
          customIdentifier = "obucks10",
          senderName = "John Doe",
          recipientEmail = "anyone@email.com",
          recipientPhoneDetails = new {
            countryCode = "NG",
            phoneNumber = "8031934751"
          }
      });

      var message = new HttpRequestMessage(HttpMethod.Post, "https://giftcards/reloadly.com/orders") {
        Content = new StringContent(json, Encoding.UTF8, "application/json")
      };

      message.Headers.TryAddWithoutValidation("Authorization", "Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");
      message.Headers.TryAddWithoutValidation("Accept", "application/com.reloadly.giftcards-v1+json");

      using
      var httpClient = new HttpClient();
      var response = await httpClient.SendAsync(message);
      var responseBody = await response.Content.ReadAsStringAsync();
      var result = JsonConvert.DeserializeObject < dynamic > (responseBody);

      Console.WriteLine(result);
    }

  }
}

File: openapi/code_samples/python/giftcards/order_a_gift_card/code.py
import requests
import json

url = "https://giftcards.reloadly.com/orders"

payload = json.dumps({
  "productId": 120,
  "countryCode": "US",
  "quantity": 1,
  "unitPrice": 1,
  "customIdentifier": "obucks10",
  "senderName": "John Doe",
  "recipientEmail": "anyone@email.com",
  "recipientPhoneDetails": {
    "countryCode": "NG",
    "phoneNumber": "8031934751"
  }
})
headers = {
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Content-Type': 'application/json',
  'Accept': 'application/com.reloadly.giftcards-v1+json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)